README.md

```diff
@@ -1,39 +1,40 @@
 # Springboot Migration Workshop
 
-This is a collection of adventures for automatic remediation based on OpenRewrite.
-Each adventure represent a valid option to apply a SpringBoot migration.
-It is not expected that you complete all our adventures, but enjoy with the ones 
-that are interesting to you because of the technologies that are applied. 
+In this repository, you'll find a variety of "adventures" that demonstrate the
+different ways you can automatically migrate or upgrade an application to Spring
+Boot 3. The core technology behind each of these adventures is
+[OpenRewrite](https://github.com/openrewrite/rewrite).
 
-For all the adventures we are going to use different solutions to migrate 
-an old version of the [spring-petclinic](https://github.com/spring-projects/spring-petclinic/)
-to SpringBoot 3.
+You do not need to complete all of these adventures. Instead, please pick the
+ones that are more interesting or relevant to your needs.
+
+Regardless of which adventure you pick, you will migrate an old version of the
+[Spring PetClinic](https://github.com/spring-projects/spring-petclinic/)
+repository to Spring Boot 3.
 
 ## Introduction
 
 [Spring Boot 3 is here; Where are you?](https://docs.google.com/presentation/d/1onKVe3tsCm9wPWMQ0i4fLDfM8vpQHyXH/edit?usp=sharing&ouid=106870370074198362402&rtpof=true&sd=true)
 
 ## Requirements
 
-In complete our adventures, you will need some basic Spring and Git knowlege and:
-
-* Java 17 and Java 8
-
-* IDE of your choice, but Visual Studio Code for the Spring Tools Adventure. 
+To complete our adventures, you will need:
 
-* GitHub account
+* Some basic Spring and Git knowledge
+* To be familiar with Java 8 and Java 17
```

Comment:
Java 17 and 8 need to be installed. I update it since the PR has been already merged.